app/controllers/task_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# Task controller
```

Comment:
Hint: Some of the `cops` from rubocop can be disabled or configured for only some paths, worth to read: https://docs.rubocop.org/rubocop/configuration.html

Reply:
In this case commenting class doesn't add any value, so it can be safely disabled